examples/tree/plot_decision_tree.py

```diff
@@ -0,0 +1,34 @@
+"""
+================================================================
+Plot the decision tree
+================================================================
+
+Plot the decision tree trained on some features of the iris dataset.
+
+See :ref:`decision tree <tree>` for more information on the estimator.
+"""
+print(__doc__)
+
+import graphviz
```

Comment:
CI will choke on this as it doesn't have graphviz. We should probably install it on CI to have the example run, but catch the import here and provide a nicer error message to the user.